### PR TITLE
Issue #6207: Add Xpath Regression Test for IllegalToken

### PR DESCRIPTION
**Pull request by Binita-tech**
Partial for Issue: #6207

Add xpath regression test for IllegalToken

### CODE REVIEW

**Comment by Binita-tech** [comment]
expected: C:\Users\Dell\Desktop\G_Checkstyle\checkstyle\src\it\resources\org\checkstyle\suppressionxpathfilter\illegaltoken\SuppressionXpathRegressionIllegalToken2.java:4:10: Using '{0}' is not allowed.
but was : Audit done.
I am getting this error, please guide.

**Comment by romani** [comment]
you do too broad suppression
```
final List<String> expectedXpathQueries = Collections.singletonList(
               "/COMPILATION_UNIT"
```
so you do no have any violation , but you expect them in Input file.

File: src/it/resources/org/checkstyle/suppressionxpathfilter/illegaltoken/SuppressionXpathRegressionIllegalToken2.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package org.checkstyle.suppressionxpathfilter.illegaltoken;
+
+public class SuppressionXpathRegressionIllegalToken2 {
+  public native void myTest(); //warn
```
Comment (nmancus1) [inline src/it/resources/org/checkstyle/suppressionxpathfilter/illegaltoken/SuppressionXpathRegressionIllegalToken2.java:4]: Please read check documentation and update test module config.

Comment (Binita-tech) [reply src/it/resources/org/checkstyle/suppressionxpathfilter/illegaltoken/SuppressionXpathRegressionIllegalToken2.java]: Done

**Review by romani — APPROVED** [review]
Ok to merge

**Review by nick-mancuso — APPROVED** [review]

**Pull request merged**